templates/CMakeLists.txt.template

```diff
@@ -250,12 +268,6 @@
       return '\n'.join(lines)
     return _func
 
-  def add_dll_annotation_macros(lib, prefix):
-    return """set_target_properties(%s PROPERTIES DEFINE_SYMBOL "%s_DLL_EXPORTS")
```

Comment:
Also, I'm not 100% sure how thing still work after removing
```
set_target_properties(%s PROPERTIES DEFINE_SYMBOL "%s_DLL_EXPORTS")
```
That had been intentional to make it less reliant on CMake magic (which creates `<target>_exports` automatically AFAIR), and more explicit overall. However, since the symbols being to switch between import/export in `port_platform.h` haven't been renamed, I'm less sure what's happening. Normally, I'd say the build would have to fail, but so far it compiled... 🤷